components/table/table-col-sort-button.js

```diff
@@ -51,11 +52,14 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 			button:disabled {
 				opacity: 0.5;
 			}
-			button:hover,
-			button:focus {
-				outline-style: none;
+			button:hover {
 				text-decoration: underline;
 			}
+			button:focus-visible {
```

Comment:
I'm not using the mixin on purpose, which means Safari <15.4 won't work. We're OK with that at this point.